tasks/managed/apply-mapping/apply-mapping.yaml

```diff
@@ -22,22 +22,112 @@ spec:
       type: string
       description: Fail the task if the resulting snapshot contains 0 components
       default: "false"
+    - name: ociStorage
+      description: The OCI repository where the Trusted Artifacts are stored.
+      type: string
+      default: "empty"
+    - name: ociArtifactExpiresAfter
+      description: Expiration date for the trusted artifacts created in the
+        OCI repository. An empty string means the artifacts do not expire
+      type: string
+      default: "1d"
```

Comment:
I suppose these artifacts are temporary, correct? Or would them be reused in case of pipeline failing? Asking because if they are not reused somehow, 1d might be too much. WDYT.

Reply:
correct. temporary. I choose `1d` to allow someone to inspect the artifacts *if* (because of timezone) the pipeline happened overnight, etc

Reply:
got it. resolving.